app/Http/Controllers/WaterWorksApprovalController.php

```diff
@@ -5,15 +5,33 @@
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Validator;
 use App\Models\Service;
+use App\Services\ServicesFromKeyword;
+use Illuminate\Pagination\Paginator;
 
 class WaterWorksApprovalController extends Controller
 {
     public function index()
     {
         $services = Service::where('status', 'pending_waterworks_inspection')->paginate(20);
-        return view('pages.waterworks-request-approval', ['route' => 'admin.waterworks-request-approvals', 'search_heading' => 'SEARCH REQUEST', 'services' => $services]);
+        return view('pages.waterworks-request-approval', ['search_route' => 'admin.water.search','route' => 'admin.waterworks-request-approvals', 'search_heading' => 'SEARCH REQUEST', 'services' => $services]);
```

Comment:
Same here, we can create a function that will build the data we provide to the view.